lib/CleanerPlugin.js

```diff
@@ -0,0 +1,41 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Tobias Koppers @sokra
+*/
+"use strict";
+const del = require("del");
```

Comment:
My only concern is based on compiler.outputFileSystem sometimes users will be using in memory build and we should in some way maybe be aware of this if using this plugin. 

Also, between available packages have we evaluated the amount and cost of transient dependencies (and maybe compare with rimraf, etc)

Reply:
@TheLarkInn wait implement https://github.com/mrmlnc/fast-glob/issues/106, new `globby` use `fast-glob` and it is allow use own file system